roomserver/state/v1/state_test.go

```diff
@@ -1,58 +0,0 @@
-// Copyright 2017 Vector Creations Ltd
```

Comment:
I don't see where these tests have landed. Where are they now?

Reply:
They're gone forever! (I accidentally forgot to resurrect them, they're back now.)